### PR TITLE
Cleanups and modernization in pvt classes

### PR DESCRIPTION
**Pull request by akva2**

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: opm/material/fluidsystems/blackoilpvt/WetGasPvt.hpp [hunk 1]
```diff
@@ -706,7 +704,7 @@ class WetGasPvt
 private:
     void updateSaturationPressure_(unsigned regionIdx)
     {
-        typedef std::pair<Scalar, Scalar> Pair;
+        using Pair = std::pair<Scalar, Scalar>;
```
Comment (blattms) [inline opm/material/fluidsystems/blackoilpvt/WetGasPvt.hpp:707]: Can't help it as the regular person for this is not here :wink:.
Do we even need this? What about just using this below
```
-            Pair val(Rv, pSat);
-            pSatSamplePoints.push_back(val);
+            pSatSamplePoints.emplace_back(Rv, pSat);
```

File: opm/material/fluidsystems/blackoilpvt/WetHumidGasPvt.hpp [hunk 2]
```diff
@@ -828,7 +826,7 @@ class WetHumidGasPvt
 private:
     void updateSaturationPressure_(unsigned regionIdx)
     {
-        typedef std::pair<Scalar, Scalar> Pair;
+        using Pair = std::pair<Scalar, Scalar>;
```
Comment (blattms) [inline opm/material/fluidsystems/blackoilpvt/WetHumidGasPvt.hpp:829]: skip and use emplace_back, below?

File: opm/material/fluidsystems/blackoilpvt/WetGasPvt.hpp (hunk 1 above)
Comment (akva2) [reply opm/material/fluidsystems/blackoilpvt/WetGasPvt.hpp]: okay. Pair is also used in lambda parameters below, so i changed to a variadic lambda there.

File: opm/material/fluidsystems/blackoilpvt/WetHumidGasPvt.hpp (hunk 2 above)
Comment (akva2) [reply opm/material/fluidsystems/blackoilpvt/WetHumidGasPvt.hpp]: ditto.

**Comment by akva2** [comment]
jenkins build this please

**Pull request merged**